lib/std/array_list.zig

```diff
@@ -50,6 +50,19 @@ pub fn ArrayListAligned(comptime T: type, comptime alignment: ?u29) type {
             self.allocator.free(self.allocatedSlice());
         }
 
+        pub usingnamespace if (T != u8) struct { } else struct {
+            const WriteReturn = @typeInfo(@TypeOf(write)).Fn.return_type.?;
+            pub const Error = @typeInfo(WriteReturn).ErrorUnion.error_set;
```

Comment:
No need to be fancy, we know its just `OutOfMemory`

Reply:
ok